core/src/calleeFunctions/CurveLpTokenUniv3Callee.ts

```diff
@@ -12,10 +12,10 @@ const getCalleeData = async function (
     collateral: CollateralConfig,
     profitAddress: string
 ): Promise<string> {
-    if (collateral.exchange.callee !== 'CurveLpTokenUniv3Callee') {
+    if (!('route' in collateral.exchanges['Curve Token V3'])) {
```

Comment:
Again, please prefer to validate `callee` name, not the key. The key is only used to be displayed on the frontend, is it not to identify the callee name. For example, **the same callee will be used multiple times** (with static vs auto routing) in the future! So you need to provide `marketId` to the `getCallee` data and validate it's callee name to be valid